packages/components/src/components/contract-card/contract-card-items/__tests__/vanilla-options-card-body.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import VanillaOptionsCardBody from '../vanilla-options-card-body';
+import { TVanillaOptionsCardBodyProps } from '../vanilla-options-card-body';
+
+describe('VanillaOptionsCardBody', () => {
+  const mock_props: TVanillaOptionsCardBodyProps = {
+    contract_info: {
+      buy_price: 100,
+      bid_price: 105,
+      entry_spot_display_value: '110',
+      barrier: '120',
+      sell_price: 95,
+      profit: -5,
+      status: 'loss',
+    },
+    currency: 'USD',
+    getCardLabels: () => ({
+      CONTRACT_VALUE: 'Contract Value',
+      ENTRY_SPOT: 'Entry Spot',
+      PURCHASE_PRICE: 'Purchase Price',
+      STRIKE: 'Strike',
+      TOTAL_PROFIT_LOSS: 'Total Profit/Loss',
+    }),
+    is_sold: true,
+    progress_slider: null,
+    status: 'loss',
+  };
+  it('should render the correct content for a sold contract', async () => {
+    // Render the component with the provided props
+    const { getByText, getByTestId }: RenderResult = render(
+        <VanillaOptionsCardBody {...mock_props}/>
+    );
+
+    const indicative_movement = getByTestId('dc-contract-card__indicative--movement');
+
+    // Test that the correct elements are present in the component
+    expect(getByText('Contract Value')).toBeInTheDocument();
+    expect(getByText('Entry Spot')).toBeInTheDocument();
+    expect(getByText('Purchase Price')).toBeInTheDocument();
+    expect(getByText('Strike')).toBeInTheDocument();
+    expect(getByText('Total Profit/Loss')).toBeInTheDocument();
```

Comment:
```suggestion
    expect(getByText(mock_props. getCardLabels().CONTRACT_VALUE)).toBeInTheDocument();
    expect(getByText(mock_props. getCardLabels().ENTRY_SPOT)).toBeInTheDocument();
    expect(getByText(mock_props. getCardLabels().PURCHASE_PRICE)).toBeInTheDocument();
    expect(getByText(mock_props. getCardLabels().STRIKE)).toBeInTheDocument();
    expect(getByText(mock_props. getCardLabels().TOTAL_PROFIT_LOSS)).toBeInTheDocument();
```
@akmal-deriv just a nit-pick for further maintainability :) same for lines 64-68